.github/workflows/debian_11.yml

```diff
@@ -0,0 +1,77 @@
+name: debian_11
+
+on: [push, pull_request]
+
+env:
+  CI_MAKE: make -f .gitlab.mk
+
+jobs:
+  # By default jobs on push and pull_request filters run duplicating each other.
+  # To avoid of it used additional external module 'skip-duplicate-actions'.
+  # Used option 'concurrent_skipping' to skip duplicating jobs. Check info:
+  # https://github.com/marketplace/actions/skip-duplicate-actions#concurrent_skipping
+  pre_job:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          # avoid of skipping previous jobs for outdated commits
+          cancel_others: 'false'
+          # avoid of duplicating runs on "push & pull_request"
+          concurrent_skipping: 'same_content'
+
+  debian_11:
+    needs: pre_job
+    if: |
+      needs.pre_job.outputs.should_skip != 'true' ||
+      github.event_name == 'push'
```

Comment:
I don't understand. Are we want to run push jobs always and skip PR jobs, when they are the same as a push job? Why not to do that in a [single line](https://github.com/h2o/h2o/commit/1c79d79e8b7686145975bc1cb04201e78b875924) without any complicated logic and external actions?

Reply:
Ok, let's try to use it instead of module, PR #5666 created for it before this PR.

Reply:
PR #5666 committed.